vision/google/cloud/vision_v1.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.cloud.gapic.vision.v1 import image_annotator_client as iac
+from google.cloud.gapic.vision.v1 import enums
+
+from google.cloud.proto.vision.v1 import geometry_pb2 as geometry
+from google.cloud.proto.vision.v1 import image_annotator_pb2 as image_annotator
```

Comment:
I would like to, but I have not had time to experiment with it yet. Same applies to video intelligence.